### PR TITLE
[helm/tf/aws] add monitoring/logging option to aptos-node

### PR DESCRIPTION
**Pull request by sherry-x**
- Add option to install monitoring chart for aws terraform
- Cleanup monitoring config a bit more

### CODE REVIEW

File: terraform/aptos-node/aws/kubernetes.tf [hunk 1]
```diff
@@ -155,6 +155,68 @@ resource "helm_release" "validator" {
...
+
+resource "helm_release" "monitoring" {
+  count       = var.enable_monitoring ? 1 : 0
+  name        = "${local.workspace_name}-mon"
+  chart       = "${path.module}/../../helm/monitoring"
+  max_history = 10
+  wait        = false
+
+  values = [
+    jsonencode(var.monitoring_helm_values),
```
Comment (rustielin) [inline terraform/aptos-node/aws/kubernetes.tf:195]: same as above

File: terraform/aptos-node/aws/kubernetes.tf [hunk 2]
```diff
@@ -155,6 +155,68 @@ resource "helm_release" "validator" {
...
+    jsonencode({
+      chain = {
+        name = var.chain_name
+      }
+      validator = {
+        name = var.validator_name
+        releaseName = local.workspace_name
```
Comment (rustielin) [inline terraform/aptos-node/aws/kubernetes.tf:202]: what is releaseName? We should try not to introduce too many mandatory configurations if it's confusing to operators

File: terraform/helm/monitoring/files/prometheus.yml [hunk 3]
```diff
@@ -18,6 +18,22 @@ rule_files:
 {{- end }}
 
 scrape_configs:
+- job_name: 'validator'
```
[the remark this answers is not in the document]
Comment (sherry-x) [reply terraform/helm/monitoring/files/prometheus.yml:21]: Ah yeah forgot about those, thought we can make things easier to read here, will change back to scrap based on annotation then...

File: terraform/aptos-node/aws/kubernetes.tf (hunk 2 above)
Comment (sherry-x) [reply terraform/aptos-node/aws/kubernetes.tf:202]: nice catch, it was testing stuff, we don't need it

File: terraform/helm/monitoring/values.yaml [hunk 4]
```diff
@@ -2,9 +2,10 @@ chain:
   name:
 validator:
   name:
+  releaseName:
```
Comment (rustielin) [inline terraform/helm/monitoring/values.yaml:5]: what is this for?

**Review by rustielin — APPROVED** [review]
Generally LGTM (didn't test this myself yet). One final question above, but stamping to unblock

File: terraform/helm/monitoring/values.yaml (hunk 4 above)
Comment (sherry-x) [reply terraform/helm/monitoring/values.yaml:5]: uhh another testing leftover...thanks for catching it

**Pull request merged**